plugins/plugin-chart-echarts/src/Timeseries/EchartsTimeseries.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { useEffect, createRef } from 'react';
+import styled from '@superset-ui/style';
+import echarts from 'echarts';
+import { EchartsTimeseriesProps } from './types';
+
+interface EchartsLineStylesProps {
+  height: number;
+  width: number;
+}
+
+const Styles = styled.div<EchartsLineStylesProps>`
+  height: ${({ height }) => height};
+  width: ${({ width }) => width};
+`;
+
+export default function EchartsTimeseries(props: EchartsTimeseriesProps) {
```

Comment:
Can we DRY and reuse the same rendering component for all chart types?

Reply:
Good idea